docs/data-management.rst

```diff
@@ -508,6 +514,17 @@ Step-by-Step
     duplicates, the link might be deleted during the subsequent
     move of that duplicate to another group.
 
+.. _Shares:
+
+Shares (discontinued feature)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously created Shares can still be viewed in the ``Shares`` tab |image6| above the left-hand side pane of OMERO.web. Nevertheless, the Shares feature is discontinued. **It is not posssible to use the** ``World`` icon |image7| above the left-hand pane in OMERO.web to create new Shares anymore. Shares have been deprecated in OMERO 5.3.0 and are discontinued in OMERO.web xxx and later. You need to use the OMERO standard permissions to share Images, by :ref:`moving the data<Movedowners>` into the appropriate group. If you also want to retain the Images in their current group, you can first duplicate them as described in :ref:`Duplicate feature<Duplicatecli>`. Nevertheless, this :ref:`Duplicate<Duplicatecli>` and :ref:`Move<Movedowners>` workflow has following limitations:
```

Comment:
OMERO.web xxx is 5.9.0.

Reply:
It was actually OMERO.server 5.6.1 that 'broke' shares (meant you couldn't see users who weren't in your own groups). But probably don't need that detail here.

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/19/commits/7be0a686118f8bab622892e64b84982ae5cfd4e9